### PR TITLE
added textdomains

### PR DESCRIPTION
**Pull request by schubi2**

### CODE REVIEW

File: library/packages/test/slide_show_test.rb [hunk 1]
```diff
@@ -150,6 +150,7 @@
 
   describe "#Redraw" do
     before do
+      textdomain "base"
```
Comment (jreidinger) [inline library/packages/test/slide_show_test.rb]: why it is needed? It should be not needed as there is not localized string ( and should not in test ).

Comment (schubi2) [reply library/packages/test/slide_show_test.rb]: RuntimeError:
No textdomain configured in Yast::SlideShowClass, cannot translate "%s Release Notes"
in OSC. Up to now I have not found the right way.....

Comment (jreidinger) [reply library/packages/test/slide_show_test.rb]: this is for sure wrong, as it is already in main https://github.com/yast/yast-yast2/blob/master/library/packages/src/modules/SlideShow.rb#L123

Comment (jreidinger) [reply library/packages/test/slide_show_test.rb]: let me check it.

Comment (jreidinger) [reply library/packages/test/slide_show_test.rb]: can you point me to log where it failed? or can I reproduce it locally in your branch with rake osc:build?

**Pull request merged**